### ISSUE TITLE
Allow editing the body/alt text when sending media events

### ISSUE THREAD

**Issue by progval** [vector-im/element-web#22100]
#### What would you like to do?

When uploading a media such as an image or audio clip, I would like to edit the `body` to be a description of the media, instead of the file name

#### Why would you like to do it?

The `body` of `m.image` is described by the spec as:

> A textual representation of the image. This could be the alt text of the image, the filename of the image, or some kind of content description for accessibility e.g. ‘image attachment’.

However, filenames are often useless to readers, and senders should be allowed (and even encouraged) to write a description for accessibility. This would allow people with disabilities or with a client that cannot inline an image preview to get an idea of what the media is about.

#### How would you like to achieve it?

After uploading the media to the server but before sending the event, display a text box to allow users to write the description. It could default to the file name to default to the current behavior

**Comment by t3chguy** [comment]
Closing in favour of https://github.com/matrix-org/matrix-spec/issues/883

**Comment by progval** [comment]
I don't think they are the same thing; the feature I described is already possible without a spec change

**Comment by t3chguy** [comment]
Unfortunately it pretty much is assumed by most clients to the the filename. I think the spec is over-hopeful there.
